src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -401,13 +413,19 @@ void ICACHE_RAM_ATTR SX127xDriver::GetLastPacketStats()
   LastPacketRSSI += negOffset;
 }
 
-void ICACHE_RAM_ATTR SX127xDriver::SetMode(SX127x_RadioOPmodes mode)
-{ //if radio is not already in the required mode set it to the requested mod
-  if (currOpmode != mode)
-  {
-    hal.writeRegister(SX127X_REG_OP_MODE, mode);
-    currOpmode = mode;
-  }
+void ICACHE_RAM_ATTR SX127xDriver::SetMode(SX127x_RadioOPmodes mode, SX12XX_Radio_Number_t radioNumber)
+{
+  /*
+  Comment out since it is difficult to keep track of dual radios.
+  When checking SPI it is also useful to see every possible SPI transaction to make sure it fits when required.
```

Comment:
Is this something that could / should be tied to a debug define?
Or are the additional (redundant) SPI mode messages not a big deal, and it's more helpful just to leave this in to make debugging less painful?

Reply:
Im not a fan of excess SPI traffic that introduces delay, but this makes no difference and everything is clear and easy to see on a LA trace.  Best to leave it IMHO because removing it will also require even more code to track both radios states.